crates/net/eth-wire/src/types/receipts.rs

```diff
@@ -161,4 +169,8 @@ mod test {
             }
         );
     }
+
+    fn decode_raw_receipts() {
+        todo!();
```

Comment:
@mattsse can you help me with this test? Where can I find an example raw receipt?